docs/pages/components.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import Head from 'docs/src/modules/components/Head';
+import Box from '@mui/material/Box';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemButton from '@mui/material/ListItemButton';
+import Typography from '@mui/material/Typography';
+import Divider from '@mui/material/Divider';
+import KeyboardArrowRightRounded from '@mui/icons-material/KeyboardArrowRightRounded';
+import AppHeader from 'docs/src/layouts/AppHeader';
+import AppFooter from 'docs/src/layouts/AppFooter';
+import BrandingProvider from 'docs/src/BrandingProvider';
+import Section from 'docs/src/layouts/Section';
+import PageContext from 'docs/src/modules/components/PageContext';
+import { pageToTitleI18n } from 'docs/src/modules/utils/helpers';
+import { useTranslate } from 'docs/src/modules/utils/i18n';
+import Link from 'docs/src/modules/components/Link';
+
+export default function Home() {
+  const { pages } = React.useContext(PageContext);
+  const t = useTranslate();
+  const componentPageData = pages.find(({ pathname }) => pathname === '/components');
+  return (
+    <BrandingProvider>
+      <Head
+        title="MUI: The React component library you always wanted"
```

Comment:
```suggestion
        title="Components - MUI"
```

Maybe?! Not sure if we should repeat the home page head title for this one.